### PR TITLE
Add suggestions for adding posix group in organizational unit (Closes #742)

### LINKED ISSUE #742: [doc] 4.4.3  Adding a New Group to an Organizational Unit
4.4.3  Adding a New Group to an Organizational Unit

https://documentation.suse.com/suse-caasp/4.1/single-html/caasp-admin/#_adding_a_new_group_to_an_organizational_unit

dn: cn=GROUP,ou=OU_NAME,dc=example,dc=org
changetype: add
objectClass: top
objectClass: groupOfNames
gidNumber: GROUPID
cn: GROUP

seem to need to add : objectclass: posixGroup  as following to avoid 
ldap_add: Object class violation (65)
        additional info: attribute "gidNumber" not allowed

dn: cn=GROUP,ou=OU_NAME,dc=example,dc=org
changetype: add
objectclass: top
objectclass: groupOfNames
objectclass: posixGroup
gidNumber: GROUPID
cn: GROUP

### PR DESCRIPTION
**Pull request by cclhsu**
add suggestions for add posix group in organizational unit (#742)

Signed-off-by: cclhsu <clark.hsu@suse.com>

# Make sure it's tested
*Technical writers will not always be able to verify the implementation!*

Any PR opened is assumed to have been verified by QA if not designated by comments or labels otherwise. Please make sure you have tested the changes and included any information that a user would require to use the documentation.

# Describe your changes
What is this PR about? Please briefly detail why this needs to be integrated
and where you think it should go if it's not obvious from the changes.

Add missing posix group to organization unit

# Related Issues / Projects
Please provide links to Bugzilla and other GitHub projects with your description if they are related to the changes.

Relates to:

# Enable maintainer updates
Please enable maintainer updates so we can push commits into your branch to make collaboration and reviews easier.

# Do not force push your branch
Please avoid force pushing to branches that are subject of pull requests. Force pushing breaks maintainer commits in many cases and is very hard (if not impossible) to untangle for backporting.

# Labels
Please set any (and all) appropriate labels that describe the status of the PR.

| **Label** | **Description** |
| --- | --- |
| P1 | PR should be worked on and merged as soon as possible |
| Blocked | Work can not proceed because other work has not been completed, PR can not be merged (code has not been merged but documentation is ready) |
| On-Hold | Underlying work is completed but the PR should not be merged |
| ReleaseNotes | User interaction is required after the introduction of this change and the change must be mentioned in the release notes |
| v3/v4/v4.x | Which version of the release the PR should be merged into, this can be multiple versions, please set the "Backport" label if it needs to go into a previous release |
| Needs Review | Some details of the PR are known to be incomplete and must be discussed with other engineers before merging (if possible assign reviewers or cc mention in comments), PR can not be merged |

### CODE REVIEW

File: adoc/admin-security-user-group-management.adoc [hunk 1]
```diff
@@ -64,6 +64,7 @@ ldapmodify -v -H <LDAP_PROTOCOL>://<LDAP_NODE_FQDN><LDAP_NODE_PROTOCOL> -D "<BIN
 dn: cn=GROUP,ou=OU_NAME,dc=example,dc=org
 changetype: add
 objectClass: top
+objectClass: posixGroup
```
Comment (Martin-Weiss) [inline adoc/admin-security-user-group-management.adoc]: Why do we need posixGroup? How is this related to the CaaSP DEX configuration?

Comment (cclhsu) [reply adoc/admin-security-user-group-management.adoc]: Our examples use `gidNumber`.  Without it, user will encounter errors as following:

seem to need to add : objectclass: posixGroup as following to avoid
ldap_add: Object class violation (65)
additional info: attribute "gidNumber" not allowed

Comment (Martin-Weiss) [reply adoc/admin-security-user-group-management.adoc]: So we should remove the gidNumber and not add posixGroup IMO.
(the whole posix attributes are not required or useful when using LDAP just for DEX)

Comment (innobead) [reply adoc/admin-security-user-group-management.adoc]: hmm, it seems making sense to me if they are meaningless in our case.

Comment (r0ckarong) [reply adoc/admin-security-user-group-management.adoc]: @cclhsu Can we remove gidNumber? This comment thread is the only thing blocking the merge.

**Comment by Martin-Weiss** [comment]
> 
> 
> @r0ckarong please merge the change. After reviews the document, we use gid/uid in our example. Therefore. we need to add `objectClass: posixGroup`

What is the sense of adding posixGroup in LDAP for DEX integration / CaaSP?

IMO this is out of scope for CaaSP and in case someone wants to do PAM integration / use PAM LDAP /SSSD  or similar - this is or needs to be covered in SLES and not in CaaSP.

File: adoc/admin-security-user-group-management.adoc (hunk 1 above)
Comment (cclhsu) [reply adoc/admin-security-user-group-management.adoc]: removed.

**Comment by r0ckarong** [comment]
So the last commits removed both gidNumber and posixGroup. I will merge this state now. Correct?

**Comment by cclhsu** [comment]
yes, I have verified that with change. New group can be created
